app/services/clients/stripe/stripeBankAccount.model.js

```diff
@@ -0,0 +1,42 @@
+'use strict'
+
+const Joi = require('joi')
+
+const { ValidationError } = require('../../../utils/errors')
+
+const schema = {
+  bank_account_sort_code: Joi.string().required(),
+  bank_account_number: Joi.string().required()
+}
+
+class StripeBankAccount {
+  constructor (body) {
+    const params = Object.assign({}, body)
+    const { error, value: model } = Joi.validate(params, schema, { allowUnknown: true, stripUnknown: true })
+
+    if (error) {
+      throw new ValidationError(`StripeBankAccount ${error.details[0].message}`)
```

Comment:
Good idea to use validation and explicitly throwing errors 👍 I'm not sure if we need to create an extended `ValidationError` in favour of just throwing an `Error` as there is nowhere right now that will be looking for error types. I suppose it shouldn't hurt anything. 

Questions following this will be: 
* How will these errors be caught and handled by controllers using this client

Reply:
As discussed we will use `Error`, thanks